src/main/webapp/app/core/user/settings/learner-profile/learner-profile.component.html

```diff
@@ -0,0 +1 @@
+<jhi-course-learner-profile />
```

Comment:
I do not think we need the component `LearnerProfileComponent` - why don't we just directly use `CourseLearnerProfileComponent`?

Reply:
This preempts future additions of non course specific profile data. There are changes already in development.

Reply:
I do not think that this is a good reasoning, but considering the size and how long this PR is open I guess we can keep it as it is